app/api/activitylog/activitylogMiddleware.js

```diff
@@ -4,7 +4,7 @@ import activitylog from './activitylog';
 
 const ignoredMethods = ['GET', 'OPTIONS', 'HEAD'];
 export const IGNORED_ENDPOINTS = [
-  '/api/users',
+  // '/api/users',
```

Comment:
I removed this endpoint completely from IGNORED_ENDPOINTS. The reason is, only 3 methods exists on this endpoint, POST, GET and DELETE, POST and DELETE need to be logged and GET should not be logged. Which is taken care of by `ignoredMethods` array... @konzz @mfacar I am not sure why the endpoint was included, kindly shed more light into this. Also thanks @daneryl for the help

Reply:
@grafitto I think it's fine to remove 'api/users' from the ignored endpoints, I'm not sure when it was added but as you found it is used to update users.

Reply:
good, we can remove the commented line then :)

Reply:
Done